post-install.txt

```diff
@@ -12,13 +12,17 @@ chroot /rootfs /usr/bin/git clone -b master https://github.com/openhab/openhabia
 if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; exit 1; fi
 ln -s /opt/openhabian/openhabian-setup.sh /rootfs/usr/local/bin/openhabian-config
 
-echo "[openHABian] === Start executing 'openhabian-setup.sh' ==="
+echo -n "[openHABian] Backup installation configuration..."
+cp /rootfs/boot/installer-config.txt /rootfs/opt/openhabian/installer-config.txt.netinstalled
+if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; exit 1; fi
+
+echo -n "[openHABian] === Start executing 'openhabian-setup.sh' ==="
 chroot /rootfs /bin/bash /opt/openhabian/openhabian-setup.sh unattended
 if [ $? -eq 0 ]; then
-  rm /rootfs/home/pi/openHABian-install-failed
-  touch /rootfs/home/pi/openHABian-install-successful
+  rm /rootfs/home/`ls /rootfs/home`/openHABian-install-failed
+  touch /rootfs/home/`ls /rootfs/home`/openHABian-install-successful
 else
-  touch /rootfs/home/pi/openHABian-install-failed
+  touch /rootfs/home/`ls /rootfs/home`/openHABian-install-failed
```

Comment:
But what if the username changes which this PR implements then we will have an regression bug in the codebase.